lib/scripting/post-process.ts

```diff
@@ -47,27 +107,84 @@ export function varDecl(result: [ string, null, string, string[] ]) {
  * Example: `Const test1 = 3.14, test2 = 4, test3 = "TEST", test4 = -5.2\n`
  * Result:
  * ```
- * 'Const',
- * null,
- * [ 'test1', null, '=', null, { type: 'Literal', value: 3.14 } ],
  * [
- *   [ 'test2', null, '=', null, { type: 'Literal', value: 4 } ],
- *   [ 'test3', null, '=', null, { type: 'Literal', value: 'TEST' } ],
- *   [ 'test4', null, '=', null, { type: 'Literal', value: -5.2 } ]
- * ],
- * [ [ [ '\n' ] ] ]
+ *    "Const",
+ *    null,
+ *    [
+ *       {
+ *          "type":"Identifier",
+ *          "name":"test1"
+ *       },
+ *       null,
+ *       "=",
+ *       null,
+ *       {
+ *          "type":"Literal",
+ *          "value":3.14
+ *       }
+ *    ],
+ *    [
+ *       [
+ *          {
+ *             "type":"Identifier",
+ *             "name":"test2"
+ *          },
+ *          null,
+ *          "=",
+ *          null,
+ *          {
+ *             "type":"Literal",
+ *             "value":4
+ *          }
+ *       ],
+ *       [
+ *          {
+ *             "type":"Identifier",
+ *             "name":"test3"
+ *          },
+ *          null,
+ *          "=",
+ *          null,
+ *          {
+ *             "type":"Literal",
+ *             "value":"TEST"
+ *          }
+ *       ],
+ *       [
+ *          {
+ *             "type":"Identifier",
+ *             "name":"test4"
+ *          },
+ *          null,
+ *          "=",
+ *          null,
+ *          {
+ *             "type":"Literal",
+ *             "value":-5.2
+ *          }
+ *       ]
+ *    ],
+ *    [
+ *       [
+ *          [
+ *             "\n"
+ *          ]
+ *       ]
+ *    ]
+ * ]

```

Comment:
Any particular reason you spread those out? Having one arg per line seems more readable to me, specially for stuff we don't care about like `[ [ [ '\n' ] ] ]`.

Reply:
I'd say:

- No more than one argument per line
- If the argument is longer than ~120 chars, split it on multiple lines

Basically: "As much readable as possible on as little space possible" :)

Reply:
Better, but I still would do it manually. For example we don't really need to list the whole body for nested nodes, what's important in that case is to quickly see that the 6th argument is an array of `ExpressionStatement`. If it's too complicated to list then it's probably a sign that it's too complicated to begin with (as noticed earlier with `[Identifier, Identifier[]]`).